README.mdown

```diff
@@ -39,6 +41,30 @@ curl -X POST -H "Content-Type: application/json" \
      http://127.0.0.1:8080/incoming/your-secret-token
 ```
 
+### UptimeRobot
+
+The URL is constructed as follows: `http(s)://<host>:<port>/incoming/uptimerobot/<your-secret-token>`
+
+#### Usage
+
+Send a GET request to the webhook URL with the follow query params.
+In UptimeRobot's settings, add the following webhook (the trailing ampersand
+is required):
+
+```
+http(s)://<host>:<port>/incoming/uptimerobot/<your-secret-token>?room=ops@example.com&
+```
+
+| Key | Value |
+| --- | ----- |
+| room | (string) The room/channel you want to post to. |
+
+Example:
+
+```sh
+curl http://127.0.0.1:8080/incoming/your-secret-token?room=ops@example.com&monitorFriendlyName=Test&monitorID=1234567&alertTypeFriendlyName=Down
+```
```

Comment:
I don't understand why there's a CURL example in here. It doesn't really make sense as an example for how to use the hook. I think this is just copied from the existing README, but the UptimeRobot settings field wouldn't look like this, or would it?

Perhaps "example" is just the wrong description, but I think an actual example of what someone needs to put in the settings would be more useful than one for manually sending a request.